features/unattached_commands.feature

```diff
@@ -383,19 +383,18 @@ Feature: Command behaviour when unattached
         esm-cache\.service
         """
         When I run `journalctl -o cat -u esm-cache.service` with sudo
-        Then stdout does not match regexp:
+        Then stdout does not contain substring:
         """
-        raise FetchFailedException\(\)
+        raise FetchFailedException()
         """
-        When I run `ls /var/crash/` with sudo
-        Then stdout does not match regexp:
+        Then stdout matches regexp:
         """
-        _usr_lib_ubuntu-advantage_esm_cache
+        WARNING|ubuntupro.apt|fail|\d+|Failed to fetch ESM Apt Cache item: https://esm.ubuntu.com/apps/ubuntu/dists/<release>-apps-security/InRelease|{}
```

Comment:
Just to double check, why is the function name `fail` here ?

Reply:
Inside `update_esm_caches`, an `EsmAcquireProgress` class is defined with a `fail` method that is called by the `apt` library when an error occurs. That method is where we log this line